### PR TITLE
Update to Fastify v4

### PR DESCRIPTION
**Pull request by Eomm**

### CODE REVIEW

**Comment by mcollina** [comment]
Are we sure we want to support both with the same major? Wouldn't be simpler to support only one?

**Comment by kibertoad** [comment]
I would vote in favour of upping major and dropping support for v3

File: README.md [hunk 1]
```diff
@@ -8,9 +8,10 @@ This package is responsible for compiling the application's `response` JSON sche
 
 ## Versions
 
-| `@fastify/fast-json-stringify-compiler` | `fast-json-stringify` | Default in `fastify` |
+| `@fastify/fast-json-stringify-compiler` | `fast-json-stringify` |  Supported `fastify` |
```
Comment (kibertoad) [inline README.md:11]: is extra space intentional?

**Review by kibertoad — APPROVED** [review]

**Review by Fdawgs — APPROVED** [review]

**Review by mcollina — APPROVED** [review]
lgtm

**Pull request merged**